### PR TITLE
Kill DailySavedExportNotification

### PR DESCRIPTION
**Pull request by orangejenny**
Kills https://github.com/dimagi/commcare-hq/pull/15158/

Branched off of https://github.com/dimagi/commcare-hq/pull/22051 - reviewers only need to look at the last commit, 0953e17c204435b9400a0b0baf514cb383491cea

Planning to delete [the class](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/export/models/new.py#L2611) in another PR once this one has been merged and deployed on all environments. @emord am I reasoning about that correctly? I've made mistakes in model deletions a couple of times.

@dannyroberts / @mkangia

### CODE REVIEW

File: corehq/apps/export/migrations/0006_delete_dailysavedexportnotification.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('export', '0005_datafile_blobmeta'),
+    ]
+
+    operations = [
+        migrations.DeleteModel(
+            name='DailySavedExportNotification',
```
Comment (emord) [inline corehq/apps/export/migrations/0006_delete_dailysavedexportnotification.py]: > Planning to delete the class in another PR once this one has been merged and deployed on all environments

This migration would delete the table in this PR

Comment (orangejenny) [reply corehq/apps/export/migrations/0006_delete_dailysavedexportnotification.py]: Removed in d0cd939

**Review by emord — APPROVED** [review]

**Comment by mkangia** [comment]
> Planning to delete the class in another PR once this one has been merged and deployed on all environments

Something to do with migration and class?

**Comment by orangejenny** [comment]
@mkangia I don't follow the question. Does looking at https://github.com/dimagi/commcare-hq/pull/22054 help answer it?

**Comment by mkangia** [comment]
@orangejenny just trying to find out the reason behind your comment in the summary above.

**Pull request merged**